src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -279,6 +279,13 @@ public function savePrivateNoteAction($customerId, Request $request)
                     $data['note']
                 ));
 
+                if ($request->isXmlHttpRequest()) {
```

Comment:
@matks this is something we could improve. When we need to perform "Save private note" action using both form submit (with page refresh and flash message) and ajax request, I think there should be a better way to handle it.

Reply:
How would you improve it ? Maybe 2 controller actions, in order to separate clearly the 2 usecases, as the input payload is very different and the output response is very different too

Reply:
Well, this should clearly be part of BO Http API. But I don't know if we can put it under `Controller/BackOffice/Api` or similar.